config/assets.php

```diff
@@ -55,6 +55,21 @@
         'cache' => false,
         'cache_path' => public_path('img'),
 
+        /*
+        |--------------------------------------------------------------------------
+        | Image Manipulation Defaults
+        |--------------------------------------------------------------------------
+        |
+        | You may define global defaults for all manipulation parameters, such as
+        | quality, format, and sharpness. These can and will be be overwritten
+        | on the tag parameter level as well as the preset level.
+        |
+        */
+
+        'defaults' => [
+            'quality' => 1,
```

Comment:
You want the default to be 1? Maybe we should just leave it commented out.

```suggestion
            // 'quality' => 1,
```

Reply:
Oops you already fixed this.